packages/core/src/html/CustomListIconProcessor.ts

```diff
@@ -10,14 +10,17 @@ interface EmojiData {
 
 const emojiData = emojiDictionary as unknown as EmojiData;
 
-const ICON_ATTRIBUTES = ['icon', 'i-width', 'i-height', 'i-size', 'i-class'];
+const ICON_ATTRIBUTES = ['icon', 'i-width', 'i-height', 'i-size', 'i-class', 'text', 't-size', 't-class'];
```

Comment:
i-size and t-size is not isolated for each icon unless you set each of the manually. (So if you set 1 with t-size, it will affect the whole list.)

I think this is not explained in the documentation. I would lean towards this as a bug rather than a lack of documentation.

Should we fix this or create a new ticket hmmm?